CMakeLists.txt

```diff
@@ -50,8 +60,25 @@ if(MSVC)
     if(MATERIALX_WARNINGS_AS_ERRORS)
         add_compile_options(/WX)
     endif()
+
+    # Use /Z7 to avoid having pdbs
```

Comment:
This modification may be better left as Autodesk-specific, since different users will have different needs for symbolic debugging, and I'd be interested in your team's thoughts on the advantages of this approach.

Reply:
Outside perspective: we prefer /Zi on our builds so this would conflict. If it really needs to be here, it should probably be guarded with an option that defaults OFF.

Reply:
We needed this for creating proper pdbs for release-with-debug builds we can make this optional based on whether that configuration is set. Would this work?

Reply:
This has been removed.

Reply:
Thanks Bernard!